content/applications/finance/sign.rst

```diff
@@ -70,4 +71,50 @@ Overall, to be recognized as valid, electronic signatures have to meet five crit
    The information provided here does not constitute legal advice; it is provided for general
    informational purposes only. As laws governing electronic signatures evolve rapidly, we cannot
    guarantee that the information is up to date. We advise you to should contact a local attorney to
-   obtain legal advice.
\ No newline at end of file
+   obtain legal advice.
+
+.. _sign/field-types:
+
+Field Types
+===========
+
+By configuring your own *Field Types*, also known as *Signature Item Types*, you can make the
+signing process even faster for your customers, partners, and employees.
+
+To create and customize fields, activate the :ref:`developer mode <developer-mode>`. Then head to
+:menuselection:`Documents --> Configuration --> Field Types` and click on *Create*.
+
+After giving your new field a name, select one of the six *Types* available:
+
+- **Signature**: the user is prompted to enter their signature either by drawing it, automatically
+  generating it based on their name, or loading a local file (usually an image). Each subsequent
+  *Signature* field then reuses the data entered in the first field.
+- **Initial**: the user is prompted to enter their initials, similarly to the *Signature* field.
+- **Text**: the user enters any text on a single line.
+- **Multiline Text**: the user enters any text on multiple lines.
+- **Checkbox**: the user can tick a box to mark their approval or consent, for example.
+- **Selection**: the user chooses a single option between a series of options.
+
+Next, you have the option to auto-complete fields for users based on their Odoo profile information
+by using *Automatic Partner Field*. To this end, the *Name*, *Email*, *Phone*, and *Company* fields
+are preconfigured in your database.
+
+.. note::
+   Users can freely edit auto-completed fields.
+
+.. image:: sign/field-example.png
+   :align: center
+   :alt: Company field example in Odoo Sign
+
+You can then change the size of the field by editing the *Default Width* and *Default Height*,
+expressed as a ratio of 1, 1 equalling the full-page's width or height. By default, the width of new
+fields you create is set to 15% (0.150) of full-page's width, while their height is set to 1.5%
+(0.015) of a full-page's height.
+
+Next, you should write a *Tip*. Tips are displayed inside arrows on the right side of the user's
```

Comment:
```suggestion
Next, write a *Tip*. Tips are displayed inside arrows on the right-hand side of the user's
```
Given that you tips are required, it makes more sense in my head to just write a tip since it's a step that they can't skip. Feel free to ignore though